plugins/MauticCrmBundle/Integration/CrmAbstractIntegration.php

```diff
@@ -187,6 +187,18 @@ public function getApiHelper()
         return $this->helper;
     }
 
+    /**
+     * Sets the API helper.
+     *
+     * @param object $helper
+     */
+    public function setApiHelper($helper)
+    {
+        if ($helper instanceof CrmApi) {
```

Comment:
Is there a reason for this over type hinting $helper? Also, public setters used only for tests can be avoided by using partial mocks using setConstructorArgs then setMethods or setMethodsExcept(). 

`